tests/pytests/test_aggregate.py

```diff
@@ -418,13 +418,23 @@ def testReducerGeneratedAliasing(self):
         self.env.assertEqual('__generated_aliasfirst_valuetitle,by,price,desc', rv[1][2])
 
     def testIssue1125(self):
-        rv = self.env.cmd('ft.aggregate', 'games', '*',
-                          'LIMIT', 0, 20000000)
-        self.env.assertEqual(2266, len(rv))
-
+        self.env.skipOnCluster()
         # SEARCH should fail
         self.env.expect('ft.search', 'games', '*', 'limit', 0, 2000000).error()     \
                 .contains('LIMIT exceeds maximum of 1000000')
+        # AGGREGATE should fail
+        self.env.expect('ft.aggregate', 'games', '*', 'limit', 0, 2000000).error()     \
+                .contains('LIMIT exceeds maximum of 1000000')
```

Comment:
@ashtul I do not think there is a default limit to ft.aggregate right? if so, if limit is not used I will get all the results?

Reply:
Also, let's add a test for such a case